integration/spark2/src/main/scala/org/apache/spark/sql/execution/strategy/DDLStrategy.scala

```diff
@@ -115,7 +116,7 @@ class DDLStrategy(sparkSession: SparkSession) extends SparkStrategy {
           .setConfigurationToCurrentThread(sparkSession.sessionState.newHadoopConf())
         FileUtils.createDatabaseDirectory(dbName, dbLocation, sparkSession.sparkContext)
         ExecutedCommandExec(createDb) :: Nil
-      case drop@DropDatabaseCommand(dbName, ifExists, isCascade) =>
+      case drop@DropDatabaseCommand(dbName, ifExists, isCascade) if FileFactory.getCarbonFile(CarbonEnv.getDatabaseLocation(dbName, sparkSession)).exists() =>
```

Comment:
move if branch to next line

Reply:
convention is to start the if condition from the same line as the 'case' if the length is less than 100 lines. So i think this is ok.

Reply:
OK